feature/platform/transceiver/tests/zr_cd_test/zr_cd_test.go

```diff
@@ -32,6 +32,8 @@ const (
 var (
 	frequencies        = []uint64{191400000, 196100000} // 400ZR OIF wavelength range
 	targetOutputPowers = []float64{-13, -9}             // 400ZR OIF Tx power range
+	opModeFlag         = flag.Int("operational_mode", 1, "vendor-specific operational-mode for the channel")
```

Comment:
just to make it explicit, can you please call this "operationalModeFlag" in all files?